### PR TITLE
feat: normalize prefix directions in suggest

### LINKED ISSUE #16: Handle `S` vs `South` in address suggest

### PR DESCRIPTION
**Pull request by stdavis**
Closes #16

image

### CODE REVIEW

**Review by eneemann — CHANGES_REQUESTED** [review]
My regex experience is minimal, so I'll have to trust that part.  Overall I think this looks good and will be very helpful.  My only suggestion to correct the spelling of "permutation" in open_sgid.py

File: src/masquerade/providers/open_sgid.py [hunk 1]
```diff
@@ -21,6 +23,18 @@
 
 connection = psycopg2.connect(database=DATABASE, user=AGRC, password=AGRC, host=HOST)
 
+directions = ['north', 'south', 'east', 'west']
+normalize_direction_infos = []
+for direction in directions:
+    #: build a list of all of the different ways to type the direction
+    #: e.g. (no, no., nor, nor., ...)
+    permiatations = []
```
Comment (eneemann) [inline src/masquerade/providers/open_sgid.py:31]:
```suggestion
    permutations = []
```

File: src/masquerade/providers/open_sgid.py [hunk 2]
```diff
@@ -21,6 +23,18 @@
...
+    for index in range(len(direction)):
+        permiatation = direction[0:index + 1]
```
Comment (eneemann) [inline src/masquerade/providers/open_sgid.py:33]:
```suggestion
        permutation = direction[0:index + 1]
```

lines 31-36: 'permiatation' should be 'permutation'

**Review by eneemann — CHANGES_REQUESTED** [review]
I guess this is being submitted as another review...just specifying the "permutation" fix.

**Comment by stdavis** [comment]
My spell check failed me! Thanks, @eneemann.

**Pull request merged**